tayo-blog/src/Pages/Newpost.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { APIURL } from '../url';
+export const Newpost = () => {
+  const [details, setDetails] = useState({
+    name: '',
+    title: '',
+    content: '',
+  });
+
+  const set = (name) => {
+    return ({ target: { value } }) => {
+      setDetails((oldValues) => ({ ...oldValues, [name]: value }));
+    };
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await axios.post(APIURL + '/', {
+        name: details.name,
+        title: details.title,
+        content: details.content,
+      });
```

Comment:
You can just pass `details` instead of going `{name: details.name, title: details.title, content: details.content}`